docs/faq/perf.md

```diff
@@ -101,88 +106,90 @@ We suggest always checking to make sure that a recent cuDNN version is used.
 
 Setting the environment `export MXNET_CUDNN_AUTOTUNE_DEFAULT=1` sometimes also helps.
 
-We show results when using various GPUs including K80 (EC2 p2.2xlarge), M40,
-and P100 (DGX-1).
+We show results when using various GPUs including K80 (EC2 p2.2xlarge), M60 (EC2 g3.4xlarge),
+and V100 (EC2 p3.2xlarge).
 
 ### Scoring results
 
 Based on
 [example/image-classification/benchmark_score.py](https://github.com/dmlc/mxnet/blob/master/example/image-classification/benchmark_score.py)
-and MXNet commit `0a03417`, with cuDNN 5.1
+and  [MXNet-1.2.0.rc1](https://github.com/apache/incubator-mxnet/releases/download/1.2.0.rc1/apache-mxnet-src-1.2.0.rc1-incubating.tar.gz), with cuDNN 7.0.5
 
 - K80 (single GPU)
 
-  | Batch | Alexnet | VGG | Inception-BN | Inception-v3 | Resnet 50 | Resnet 152 |
-  | --- | --- | --- | --- | --- | --- | --- |
-  |   1 | 202.66  | 70.76 | 74.91  | 42.61  | 70.94 | 24.87 |
-  |   2 | 233.76  | 63.53 | 119.60  | 60.09  | 92.28 | 34.23 |
-  |   4 | 367.91  | 78.16 | 164.41  | 72.30  | 116.68 | 44.76 |
-  |   8 | 624.14  | 119.06 | 195.24  | 79.62  | 129.37 | 50.96 |
-  |  16 | 1071.19 | 195.83 | 256.06  | 99.38  | 160.40 | 66.51 |
-  |  32 | 1443.90 | 228.96 | 287.93  | 106.43  | 167.12 | 69.73 |
-
-- M40
-
-  | Batch | Alexnet | VGG | Inception-BN | Inception-v3 | Resnet 50 | Resnet 152 |
-  | --- | --- | --- | --- | --- | --- | --- |
-  |   1 | 412.09 | 142.10 | 115.89  | 64.40  | 126.90 | 46.15 |
-  |   2 | 743.49 | 212.21 | 205.31  | 108.06  | 202.17 | 75.05 |
-  |   4 | 1155.43 | 280.92 | 335.69  | 161.59  | 266.53 | 106.83 |
-  |   8 | 1606.87 | 332.76 | 491.12  | 224.22  | 317.20 | 128.67 |
-  |  16 | 2070.97 | 400.10 | 618.25  | 251.87  | 335.62 | 134.60 |
-  |  32 | 2694.91 | 466.95 | 624.27  | 258.59  | 373.35 | 152.71 |
-
-- P100
-
-  | Batch | Alexnet | VGG | Inception-BN | Inception-v3 | Resnet 50 | Resnet 152 |
-  | --- | --- | --- | --- | --- | --- | --- |
-  |   1 | 624.84 | 294.6 | 139.82  | 80.17  | 162.27 | 58.99 |
-  |   2 | 1226.85 | 282.3 | 267.41  | 142.63  | 278.02 | 102.95 |
-  |   4 | 1934.97 | 399.3 | 463.38  | 225.56  | 423.63 | 168.91 |
-  |   8 | 2900.54 | 522.9 | 709.30  | 319.52  | 529.34 | 210.10 |
-  |  16 | 4063.70 | 755.3 | 949.22  | 444.65  | 647.43 | 270.07 |
-  |  32 | 4883.77 | 854.4 | 1197.74  | 493.72  | 713.17 | 294.17 |
+| Batch | Alexnet | VGG    | Inception-BN | Inception-v3 | Resnet 50 | Resnet 152 |
+|-------|---------|--------|--------------|--------------|-----------|------------|
+| 1     | 243.93  | 43.59  | 68.62        | 35.52        | 67.41     | 23.65      |
```

Comment:
Seems like we're having a few regressions :(

Reply:
First, @mli told me that M60 is supposed to be slower than M40.
VGG is slower because a different model was used. The original performance was measured a long time ago. Since then, the implementation of VGG has changed. The current version of VGG has many more layers. If you want to know more details, I think @TaoLv can tell you more.

Reply:
Right. `benchmark_score.py` was changed last December ([commit](https://github.com/apache/incubator-mxnet/commit/85a5b8ed9af5bc312bdcb74d35b0693933e91a2b#diff-8f8c9a6af71eaf8a0e1a91523c37b260)) and VGG test was updated from VGG-11 to VGG-16. Perf numbers in this PR are measured on VGG-16 and previous perf numbers of MKLML were measured on VGG-11.

Reply:
Actually, this change in VGG applies to all benchmark results. Not just MKLML vs. MKLDNN.